contracts/interfaces/IAxelarGMPGateway.sol

```diff
@@ -0,0 +1,84 @@
+// SPDX-License-Identifier: MIT
+
+pragma solidity ^0.8.0;
+
+import { IGovernable } from './IGovernable.sol';
+import { IImplementation } from './IImplementation.sol';
+
+interface IAxelarGMPGateway is IImplementation, IGovernable {
```

Comment:
```suggestion
interface IAxelarGMPGateway {
```
Also not relevant to the core API. Differentiating between the interface needed for the full implementation and a GMP executable